sources/Outputs/src/Dyd.cpp

```diff
@@ -161,7 +161,7 @@ Dyd::writeHvdcLine(const algo::HvdcLineDefinition& hvdcLine, const std::string&
   }
 
   return model;
-}  // namespace outputs
```

Comment:
Why removing this?

Reply:
Formatting error which added a comment, this braket does not correspond to a namespace